src/main/java/com/gmail/val59000mc/listeners/PlayerDeathListener.java

```diff
@@ -58,10 +62,21 @@ public void onPlayerDeath(PlayerDeathEvent event){
 
 			if(cfg.getEnableKillEvent()){
 				double reward = cfg.getRewardKillEvent();
-				VaultManager.addMoney(killer, reward);
-				if(!Lang.EVENT_KILL_REWARD.isEmpty()){
-					killer.sendMessage(Lang.EVENT_KILL_REWARD.replace("%money%", ""+reward));
+				List<String> killCommands = cfg.getKillCommands();
+				if (reward > 0) {
+					VaultManager.addMoney(killer, reward);
+					if (!Lang.EVENT_KILL_REWARD.isEmpty()) {
+						killer.sendMessage(Lang.EVENT_KILL_REWARD.replace("%money%", "" + reward));
+					}
 				}
+				// If the list is empty, this will never execute
+				killCommands.forEach(cmd -> {
+					try {
+						Bukkit.dispatchCommand(Bukkit.getConsoleSender(), cmd.replace("%name%", uhcKiller.getRealName()));
+					} catch (CommandException exception){
+						Bukkit.getLogger().warning("The command: '" + cmd + "' does not exists.");
```

Comment:
This exception also gets thrown when there is an error in the command, so I'd change this message to:
```
Bukkit.getLogger().warning("[UhcCore] Failed to execute kill reward command: " + cmd);
```
And also always print exceptions to console so add
```
exception.printStackTrace();
```